docs/modules/ROOT/pages/api/erc721.adoc

```diff
@@ -225,6 +225,18 @@ NOTE: Implementing xref:api/introspection.adoc#SRC5Component[SRC5Component] is a
 * xref:#ERC721-tokenURI[`++tokenURI(self, tokenId)++`]
 --
 
+[.contract-index#ERC721Component-Embeddable-Mixin-Impl]
+.Embeddable Mixin implementation
+
+--
+.ERC721MixinImpl
+* xref:#ERC721Component-Embeddable-Impls[`++ERC721Impl++`]
+* xref:#ERC721Component-Embeddable-Impls[`++ERC721MetadataImpl++`]
+* xref:#ERC721Component-Embeddable-Impls-camelCase[`++ERC721CamelOnlyImpl++`]
+* xref:#ERC721Component-Embeddable-Impls-camelCase[`++ERC721MetadataCamelOnlyImpl++`]
+* xref:api/introspection.adoc#SRC5Component-Embeddable-Impls[`++SRC5Impl++`]
```

Comment:
I like this approach, a bit "linky", since it requires a lot of jumping and navigation, but at least it's concise.
what I'm not that sure about is how it sits between the external impls and the internal

now that we have this new category I think it might make sense to merge the snake block with the camel one, the subheadings should be enough separation.

before we cared about it to be able to quickly understand the regular api without too much clout, but now that we have mixins I think that's solved by saying which ones are included and then you can go to each to see the details.

Reply:
this way we'd have embeddable impls, mixins, and internal impl. maybe i'd even put mixins above, as preferred option, I think it will also look better

Reply:
the only problem i see with this is that links don't seem to be working with subheadings (i.e. specific impls within each block)